hack/validate/vendor

```diff
@@ -10,9 +10,8 @@ validate_vendor_diff(){
 
 	if [ ${#files[@]} -gt 0 ]; then
 		# Remove vendor/ first so  that anything not included in vendor.conf will
-		# cause the validation to fail. archive/tar is a special case, see vendor.conf
-		# for details.
-		ls -d vendor/* | grep -v vendor/archive | xargs rm -rf
+		# cause the validation to fail.
+		ls -d vendor/* | xargs rm -rf
```

Comment:
If we no longer need to keep the `vendor/archive` dependency; should we bump the version of `vndr` to the latest version, and remove this line? (latest version cleans the vendor dir if you do a full `vendor`)

Reply:
Will address as a followup.

Reply:
Added